iris-mpc-cpu/src/genesis/state_sync.rs

```diff
@@ -22,6 +22,9 @@ pub struct Config {
 
     // Identifier of the last modification ID to be indexed.
     pub max_modification_id: i64,
+
+    // Identifier of the max completed modification the node has seen
+    pub max_completed_modification: i64,
```

Comment:
Possibly calling this `max_completed_modification_id` would indeed make it clearer